steam_desktop_updater.py

```diff
@@ -163,7 +163,9 @@ def get_installed_apps(steam_root):
     apps = []
     logging.info('Searching library folders')
     with open(os.path.join(steam_root, 'steamapps', 'libraryfolders.vdf'), 'r') as lf:
-        for k, v in vdf.load(lf)['LibraryFolders'].items():
+        library_folders = vdf.load(lf)['LibraryFolders']
+        library_folders['0'] = steam_root
```

Comment:
Pushing items into LibraryFolders doesn't seems safe, e.g. if Steam will add item `"0"` there - we'll end up overwriting it.

Reply:
Can you at least add and assertion before assingning the value? E.g. `assert '0' not in library_folders`

Reply:
Let's handle that by building a new list.

Reply:
Too late, already merged! But yeah, a new list would be better.

Reply:
I see that you beat me to it, but there's an alternative solution at https://github.com/mavit/steam-desktop-updater/commit/807b43ae47e0a1c50a98e8495f3107e6e3ff90fa

Reply:
Don't mind sending it as an another PR?